### ISSUE TITLE
MacOS uv-qt crashes

### ISSUE THREAD

**Issue by alatteri** [CESNET/UltraGrid#49]
crashes upon launch.  command line uv works though.

```
Process:               uv-qt [67087]
Path:                  /Applications/uv-qt.app/Contents/MacOS/uv-qt
Identifier:            cz.cesnet.ultragrid.uv-qt
Version:               0
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           uv-qt [67087]
User ID:               501

Date/Time:             2020-04-06 08:31:19.252 -0700
OS Version:            Mac OS X 10.15.3 (19D76)
Report Version:        12
Bridge OS Version:     4.2 (17P3050)
Anonymous UUID:        0B67B72D-5D2F-57E9-213C-EFFEDB105E2B


Time Awake Since Boot: 210000 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/opt/qt/lib/QtWidgets.framework/Versions/5/QtWidgets
  Referenced from: /Applications/uv-qt.app/Contents/MacOS/uv-qt
  Reason: image not found

Binary Images:
       0x100935000 -        0x10096afff +cz.cesnet.ultragrid.uv-qt (0) <80FE8209-FED0-3852-8C88-AF2685A9D241> /Applications/uv-qt.app/Contents/MacOS/uv-qt
       0x10c2b3000 -        0x10c343cb7  dyld (733.8) <EBC07CB6-870A-3A8E-B48A-67F62EA161F3> /usr/lib/dyld
    0x7fff2d1d4000 -     0x7fff2d654fe7  com.apple.CoreFoundation (6.9 - 1674.114) <5810CC45-FB5D-3A78-861F-7BE03F9B2FDD> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation

```

**Comment by alatteri** [comment]
I should add this is with Nightly.  Haven't tried release.

**Comment by MartinPulec** [comment]
This should be fixed now. Thanks for reporting. I've just noticed that a webcam acquisition doesn't work with the build after signing the image but I'll take a look on that.

**Comment by alatteri** [comment]
confirmed.  And Dark Mode works too!

**Comment by MartinPulec** [comment]
>  I've just noticed that a webcam acquisition doesn't work with the build after signing the image but I'll take a look on that.

This should now be fixed as well (refers bug #53).
